terraform/cp4ba/terraform.tfvars

```diff
@@ -0,0 +1,47 @@
+###################### CLOUD ######################
+ibmcloud_api_key      = "******************"
+iaas_classic_username = "******************"
+resource_group        = "******************"
+region                = "******************"
+
+###################### LDAP ######################
+datacenter            = "dal12"
+hostname              = "ldapvm"
+ibmcloud_domain       = "ibm.cloud"
+cores                 = 2
+memory                = 4096
+disks                 = [25]
+hourly_billing        = true
+local_disk            = true
+private_network_only  = false
+
+###################### ROKS ######################
+on_vpc              = false
+entitlement         = "cloud_pak"
```

Comment:
Conversation marked as resolved, but not seeing the fix.

Reply:
Fixed